pkgs/top-level/python-packages.nix

```diff
@@ -15023,11 +15023,13 @@ let
 
   wxPython = self.wxPython28;
 
-  wxPython28 = callPackage ../development/python-modules/wxPython/2.8.nix {
+  wxPython28 = import ../development/python-modules/wxPython/2.8.nix {
```

Comment:
It is used for the real call inside, so IMO the result should be overridable. These imports are just thin wrappers.